csharp/test/Microsoft.ML.OnnxRuntime.Tests.MAUI/Microsoft.ML.OnnxRuntime.Tests.MAUI.csproj

```diff
@@ -7,7 +7,7 @@
 
     <!-- General app properties -->
     <PropertyGroup>
-        <TargetFrameworks>net8.0-android;net8.0-ios;net8.0-maccatalyst</TargetFrameworks>
+        <TargetFrameworks>net8.0;net8.0-android;net8.0-ios;net8.0-maccatalyst</TargetFrameworks>
```

Comment:
Why does the MAUI project need a net-8.0 target with an OutputType that isn't an exe? What is produced in that case and who would use it?

Reply:
Oops, I added those lines for a previous test project structure where the MAUI csproj was imported by a net8.0 project and I added those lines to make it compatible. I removed those lines in the last commit and it works fine